tests/DucksNet.IntegrationTests/AppointmentsControllerTests.cs

```diff
@@ -155,22 +214,6 @@ private async Task<Guid> SetupPet(Guid ownerId ,string petSizeString)
         return pet!.Id;
     }
 
-    private async Task<Guid> SetupCage(Guid officeId, string size)
-    {        
-        var sut = new CreateCageDto {
```

Comment:
why did you remove it?

Reply:
Because it wasn't used in the tests and it appeared as dead code in the coverage